mbed_flasher/flashers/FlasherMbed.py

```diff
@@ -160,7 +160,27 @@ def flash(self, source, target, method, no_reset):
                 return EXIT_CODE_FILE_COULD_NOT_BE_READ
 
             self.logger.debug("copy finished")
-            sleep(4)
+
+            def worker(mp):
+                mount_point_previous_mounted = None
+
+                count = 10
+
+                while count > 0:
+                    print os.path.ismount(mp)
+
+                    if os.path.ismount(mp):
+                        mount_point_previous_mounted = True
+
+                    if not os.path.ismount(mp) and mount_point_previous_mounted is True:
```

Comment:
Why does os.path.ismount() need to be False? Am I missing something?

Reply:
so this is to check if the mount point is disappeared after file copyied.

Reply:
Ah, ok.